test/A.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+var eq = require('./shared/utils').eq;
+var R = require('ramda');
+var S = require('..');
+
+describe('A', function() {
+
+  it('is a binary function', function() {
+    eq(typeof S.A, 'function');
+    eq(S.A.length, 2);
+  });
+
+  it('A(f, x) is equivalent to f(x)', function() {
+    eq(S.A(S.inc, 1), 2);
+    eq(R.map(S.A(R.__, 100), [S.inc, Math.sqrt]), [101, 10]);
+  });
+
+  it('is curried', function() {
+    eq(S.A(S.inc).length, 1);
+    eq(S.A(S.inc)(1), 2);
+  });
+
+});
```

Comment:
Hehe. No bother, I knew what I was signing up for when I offered to do this. Will be a finicky one to get right!